File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void block_height(int n)
{
    //for column
    for (int i = 0; i < n; i++)
    {
        int k = (n - 2) - i;
        //generating required no. of spaces to right align the hashes
        while (k >= 0)
        {
            printf(" ");
            k = k - 1;
        }
        //for left pyramid block
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //space of one unit between the 2 pyramid blocks
        printf(" ");
        //for right pyramid block
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int main(void)
{
    int n;
    //requiring input to be positive numbers between 0 and 9
    do
    {
        n = get_int("Height?\n");
    }
    while (n < 1 || n > 8);
    //calling the function
    block_height(n);
}

File: sentimental-mario-more/mario.py
def main():
    height = 0
    try:
        while height > 8 or height <= 0:
            height = int(input("Height: "))
    except ValueError:
        while height > 8 or height <= 0:
            height = int(input("Height: "))
    i = 1
    for i in range(height+2):
        print(" " * (height-i), end='')
        print("#" * i, end='')
        print("  ", end='')
        print("#" * i)

if __name__ == '__main__':
    main()